src/components/ContentHeader/ContentHeader.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import "./ContentHeader.scss";
+
+const ContentHeader = ({title, link}) => {
+
+  //href attribute should be changed to 'to' when routing is implemented
+
+  return (
+    <div className="content-header">
+        <h1 className="content-header__heading">{title}</h1>
+        <a className="content-header__anchor" href={link}>
```

Comment:
I think you can add in the `<Link/>` component here there is basic routing in App at the moment.